contentstore/tests/tests_server.py

```diff
@@ -0,0 +1,456 @@
+                                                      self.headers))
+        return FakeResponse(resp.code, resp.data)
+
+    def delete(self, path, data=None, format=None, content_type=None):
+        data, content_type = self._encode_data(data, format, content_type)
+        self.headers["Content-Type"] = content_type
+        resp = self.api.handle_request(self.req_class('DELETE', path, data,
+                                                      self.headers))
+        return FakeResponse(resp.code, resp.data)
+
+    def options(self, path, data=None, format=None, content_type=None):
+        data, content_type = self._encode_data(data, format, content_type)
+        self.headers["Content-Type"] = content_type
+        resp = self.api.handle_request(self.req_class('OPTIONS', path, data,
+                                                      self.headers))
+        return FakeResponse(resp.code, resp.data)
+
+
+class TestFakeContentStore(TestCase, ContentStoreApiTestMixin):
+
+    def setUp(self):
+        try:
+            from verified_fake.fake_contentstore import (Request,
+                                                         FakeContentStoreApi)
+        except ImportError as err:
+            if "verified_fake" not in err.args[0]:
+                raise
+            raise ImportError(" ".join([
+                err.args[0],
+                "(install from pypi or the 'verified_fake' directory)"]))
+
+        self.req_class = Request
+        self.api_class = FakeContentStoreApi
+        self.api = self.api_class("", "token-1", {}, {})
+        self.api.messagesets.endpoint_data = {}
+        self.api.schedules.endpoint_data = {}
+        self.api.messages.endpoint_data = {}
+        self.api.binary_contents.endpoint_data = {}
+        self.client = self.make_client()
+
+    def make_client(self):
+        client = FakeClient(self.api, self.req_class,
+                            {"Authorization": "Token token-1"})
+        return client
+
+    def make_schedule(self, minute="0", hour="1", day_of_week="*",
+                      day_of_month="*", month_of_year="*"):
+        schedule = self.api.make_schedule_dict({
+            u'minute': minute,
+            u'hour': hour,
+            u'day_of_week': day_of_week,
+            u'day_of_month': day_of_month,
+            u'month_of_year': month_of_year,
+            u'created_at': u'2014-07-25 12:44:11.159151',
+            u'updated_at': u'2014-07-25 12:44:11.159151',
+        })
+        self.api.schedules.endpoint_data[schedule["id"]] = schedule
+        return Struct(**schedule)
+
+    def get_schedule(self, schedule_id=None):
+        data = self.api.schedules.endpoint_data
+        if schedule_id is None:
+            return data.popitem()[1]
+        else:
+            return data.get(schedule_id)
+
+    def get_schedules(self):
+        data = self.api.schedules.endpoint_data
+        s = []
+        for key, schedule in data.iteritems():
+            s.append(schedule)
```

Comment:
`return data.values()`? Also, is there a reason why we use `Struct` when creating to mimic Django, but just dicts when getting?